tests/helpers/test_service.py

```diff
@@ -470,13 +475,7 @@ def test_async_get_all_descriptions(hass):
 
     assert len(mock_service_platform_call.mock_calls) == 1
     entities = mock_service_platform_call.mock_calls[0][1][2]
-    assert entities == [
-        mock_entities["light.kitchen"],
-        mock_entities["light.living_room"],
-    ]
-    assert (
-        "Not passing an entity ID to a service to target " "all entities is deprecated"
-    ) in caplog.text
+    assert entities == []
 
```

Comment:
Based on title of test, should this one use `ENTITY_MATCH_ALL`

Reply:
Sorry for leaving slightly misplaced comments, I couldn't figure out how to leave comments in parts of unchanged code.

Reply:
Still can't figure it out, in the previous test, you can use `ENTITY_MATCH_ALL` instead of `"all"`

Reply:
ENTITY_MATCH_ALL is defined as `all` :)

Reply:
My comment was around consistency, but you are right that it doesn't matter for functionality.  Just a suggestion.